### ISSUE TITLE
Add some descriptive text in each Docs API page

### ISSUE THREAD

**Issue by fred** [omise/docs#46]
We should add some extra description to each API page explaining what it is and how it can be used in practice.
For example, in https://docs.omise.co/api/recipients/, explain the purpose and how it can be used.

**Comment by ppp-master** [comment]
Yeah, that's missing from most pages. Charges has a short description ( https://docs.omise.co/api/charges/ ), so the space/styling is ready, we just need to add text.
